content/docs/manifest.json

```diff
@@ -28,28 +28,37 @@
               "path": "/docs/installation/compatibility.md"
             },
             {
-              "title": "kubectl apply",
+              "title": "a. kubectl apply",
               "path": "/docs/installation/kubectl.md"
             },
             {
-              "title": "Helm",
+              "title": "b. Helm",
               "path": "/docs/installation/helm.md"
             },
             {
-              "title": "OperatorHub (OLM)",
+              "title": "c. OperatorHub (OLM)",
               "path": "/docs/installation/operator-lifecycle-manager.md"
             },
             {
-              "title": "Other tools",
+              "title": "d. Other tools",
               "path": "/docs/installation/other-tools.md"
             },
             {
-              "title": "Verifying",
-              "path": "/docs/installation/verify.md"
+              "title": "Installation Options",
+              "routes": [
+                {
+                  "title": "Feature flags",
+                  "path": "/docs/installation/featureflags.md"
+                },
+                {
+                  "title": "Best Practice",
+                  "path": "/docs/installation/best-practice.md"
+                }
+              ]
             },
             {
-              "title": "Feature flags",
-              "path": "/docs/installation/featureflags.md"
+              "title": "Verify installation",
+              "path": "/docs/installation/verify.md"
```

Comment:
We're going to remove this page and add the verify instructions at the end of the kubectl apply page. In a future PR.